### ISSUE TITLE
RoonServer failing to start???

### ISSUE THREAD

**Issue by mtjoseph** [steefdebruijn/docker-roonserver#9]
Any ideas on how to resolve this or strategies for debugging? I'm running this under FreeNAS 11.3-U5, Docker version 19.03.11, build 42e35e61f3, Portainer v.2.0.0 (Docker manager). I've deployed the docker-roonserver image under this environment and been running for at least a year without issue. I noticed the problem only several days ago.

Docker log file shows the following:
Warn:  get lock file path: /tmp/.rnsgem0-
Trace: [childprocess] using unix child process
Initializing
Started
Not responding
Error
Initializing
Started
Not responding
...

RoonServer.log:
2/06 20:40:02 Info: Starting RoonServer v1.7 (build 667) stable on linuxx64
12/06 20:40:03 Trace: Checking if we are already running
12/06 20:40:03 Warn: get lock file path: /tmp/.rnsems0-roon
12/06 20:40:03 Trace: Nope, we are the only one running
12/06 20:40:03 Info: Is 64 bit? True
12/06 20:40:04 Trace: [orbit] init seq=0, pending=0, spcount=7, uidcount=1
12/06 20:40:05 Info: Command Line Argument: -watchdogport=36055
12/06 20:40:05 Trace: [childprocess] using .NET child process
12/06 20:40:06 Trace: [realtime] fetching time from NTP server
12/06 20:40:07 Info: [stats] 446mb Virtual, 40mb Physical, 16mb Managed, 0 Handles, 7 Threads
12/06 20:40:07 Trace: [realtime] Got time from NTP: 12/07/2020 04:40:07 (3816304807197ms)
12/06 20:40:07 Trace: [realtime] Updated clock skew to -00:00:00.4632230 (-463.223ms)
12/06 20:40:08 Info: [broker] starting 4e8c8e4a-236b-465e-b0f2-81c668b5e402
12/06 20:40:10 Trace: [httpcache] loaded 834 cache entries from /data/RoonServer/Cache/httpcache_2.db, current: 127mb / 128mb
12/06 20:40:11 Debug: [broker/filebrowser/nassharemaker] rebuildsymlinks
12/06 20:40:11 Debug: [broker/filebrowser/volumeattached] initial listing found drive mounted at /
12/06 20:40:11 Debug: [broker/filebrowser/volumeattached] initial listing found drive mounted at /music
12/06 20:40:11 Debug: [broker/filebrowser/volumeattached] skipping /music because it is not a /dev/sd[0-9]* (mountline: /dev/vda2 /music ext4 rw,relatime,stripe=4,data=ordered 0 0)

**Comment by mtjoseph** [comment]
Additional observations. When trying to install using Roon install script ./roonserver-installer-linuxx64.sh. Executing "service roonservice --full-restart" yields:

Same result.

Initializing
Started
Not responding
Error
Initializing
Started
Not responding

**Comment by mtjoseph** [comment]
I resolved it by doing a clean install of the VM on FreeNas. My best guess is filesystem nd/or docker configuration corruption. I was not able to determine specifically what the issue was.

For any one else that may find themselves in this whole. I followed these instructions for Freenas 11.3
https://blog.hugopoi.net/2020/03/01/install-rancheros-on-freenas-11-3/

Once the VM is up and running, I installed Portainer and comassky / roon-docker-server

Portainer:
docker run -d -p 9000:9000 --name portainer --restart always --network=host -v /var/run/docker.sock:/var/run/docker.sock -v portainer_data:/data portainer/portainer-ce

roon-docker-server:
docker run -d --net="host" --name roonserver --restart always -e TZ="America/Los_Angeles" -v roon-app:/app -v roon-data:/data -v roon-music:/music -v roon-backups:/backup comassky/roon:latest

Now off to do some Easy Listening with Roon!
